src/axom/quest/tests/quest_sampling_shaper.cpp

```diff
@@ -1038,6 +1052,204 @@ units: cm
   }
 }
 
+TEST_F(SamplingShaperTest2D, check_underscores)
+{
+  const auto& testname =
+    ::testing::UnitTest::GetInstance()->current_test_info()->name();
+
+  constexpr double radius = 1.5;
+
+  const std::string shape_template = R"(
```

Comment:
Probably a silly question: what is the 'R' for (here and below)?

Reply:
This type of pattern allows you do define C++ multi-line strings. 

It's a life changer for inline strings.

Reply:
They're raw string literals. In this case, they make it easy to have multiline strings without lots of extra symbols like "\n".

See: https://en.cppreference.com/w/cpp/language/string_literal

Reply:
That's cool. I was unaware. Thanks.